apps/course/src/app/features/legal/legal-overview/legal-overview.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="mx-1 md:mx-3 lg:mx-32 xl:mx-60 min-h-1-2">
+  @if (isLoading) {
+    <app-loading-bars />
+  } @else if (!isLoading) {
+    <div class="navbar bg-base-100 border-b-2">
+      <div class="items-center flex-1">
+        <button class="btn btn-ghost text-2xl btn-lg">Legal</button>
+      </div>
+    </div>
+    <div class="flex flex-col xl:flex-row flex-wrap justify-center">
+      @for (legal of legal; track $index) {
```

Comment:
Please track the legal's by `id`. `...; track legal.id)`

Reply:
Legal does not contain an ID. Should I change that so it has an ID?

Reply:
I think you can also use the `file` field instead, since it also is unique. 🔢